src/openms/source/CHEMISTRY/ProForma.cpp

```diff
@@ -303,22 +353,22 @@ namespace OpenMS
 
         if (it != modifications_.end())
         {
-          if (!it->second.modification_name.empty())
+          if (!it->second.getName().empty())
           {
-            mod_name << "[" << it->second.modification_name << "]";
+            mod_name << "[" << it->second.getName() << "]";
           }
-          else if (it->second.mass_shift != 0.0)
+          else if (it->second.getDiffMonoMass() != 0.0)
           {
-            mod_name << "[" << (it->second.mass_shift > 0 ? "+" : "") << it->second.mass_shift << "]";
+            mod_name << "[" << (it->second.getDiffMonoMass() > 0 ? "+" : "") << it->second.getDiffMonoMass() << "]";
           }
 
-          if (it->second.range.first == it->second.range.second)
+          if (it->second.getPositionRange().first == it->second.getPositionRange().second)
           {
             ss << mod_name.str();
             mod_name.clear();
           }
 
-          if (it->second.ambiguous_start)
+          if (it->second.isAmbiguous())
```

Comment:
Hmmm having this as part of a ResidueModification looks a bit fishy to me(at first glance).